### ISSUE TITLE
Jit error for static_argument  in a custom jax function

### ISSUE THREAD

**Issue by srxzr** [google/objax#204]
I implemented a custom activation layer which has a static argument, how should I pass the static_arguments to the objax.JIT ?

`
def select_topK(w,b,w_score,b_score,topk):
    values = jn.concatenate((w_score, b_score), axis=None)
    v,inds = lax.top_k(values,topk)
    thresh = v.min()
    return jn.where(w_score>thresh, w, 0), jn.where(b_score>thresh, b, 0)
`
and I have a custom layer, where I call this function :
`
def __call__(self,x):
    ...
    w,b = select_top(self.w.value,self.b.value,self.w_score.value,self.b_score.value,self.topK)
    ....

`

I can run jax.jit(select_top,static_argument=(4,)), but when I call objax.JIT on the whole model I get a jit error (concrete value is expected). I tried @jax.partial(jit, static_argnums=4) and also defining a partial function manually which didn't work. do you have any suggestions how should I pass the static_argument parameters  ?

**Comment by AlexeyKurakin** [comment]
To clarify your question:
a) are you trying to JIT `select_topK` separately from the module where it's used?
b) have you tried to run `objax.Jit` on entire module?

Just by looking at your example I suspect that you have to JIT the Objax Module and static argnum should be an argument to `__call__` function of the module. Something like following:

```python
class SomeModule(objax.Module):

    def __call__(self, x, topK):
        # ...
        w,b = select_topK(self.w.value, self.b.value, self.w_score.value, self.b_score.value, topK)
        # ...


m = SomeModule()
jit_m = objax.Jit(m, static_argnum=(1,))

y = jit_m(x, topK=5)
```
